python/setup.py

```diff
@@ -14,17 +14,12 @@
     'cython'
 ]
 
-try:
-    numpy_include = numpy.get_include()
-except AttributeError:
-    numpy_include = numpy.get_numpy_include()
-
 cython_files = ['cudf/bindings/*.pyx']
```

Comment:
I'm not sure a wild-card is the best way to aggregate the appropriate Cython files.

Reply:
It is the standard way, and out of the scope of this PR

Reply:
(not totally disagreeing with you though, not doing it that way in cuML to be fair, would like to talk outside of the PR)

Reply:
Out of scope of this PR.

Reply:
I would tend to disagree that it's out of scope of this PR. If you're making changes to the `setup.py` file in a hot-fix, then it makes sense to address relatively small adjustments for consistency with respect to other elements of our software ecosystem.

A wild-card will grab any `.pyx` file extension upon install, which is a very easy place for errors to occur. Specifying the file list avoids potential errors, and makes this more robust/intentional.

If we want to put it into another PR, that's fine, but I don't know the time we'll have reason to touch this file.

Reply:
Given we merged this already and we wanted to do something quick and dirty for a hotfix, lets restart this discussion for 0.6 along with further build improvements across both python and CMake.